protobuf/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## Unreleased
+
+* Update READMEs of `protobuf` and `protoc_plugin`:
+  * Use `dart pub` instead of `pub` in command examples (a7e75cb)
+  * Fix typos, clarify installation instructions, mention native compilation
+    for `protoc_plugin` (#610)
```

Comment:
I think adding issue numbers to the bullets is great (I have not been doing that) perhaps make them full links somehow, show they will be clickable!

Reply:
> perhaps make them full links somehow, show they will be clickable!

I did this but the process is tedious and file size will be doubled as every line has at least one link. Do you want to keep it?

Reply:
Yes, personally i think it is nice, and don't mind the file size increase.

I wonder if a vim macro or something similar could make the tedium smaller (You could also use inline links).

Do what you feel is better, issue numbers is already nice!

Reply:
OK. It will be written once and read many times so I'm fine with extra lines if it makes reading easier.

Reply:
I think it especially makes it easier in the rendered display: eg. https://pub.dev/packages/protobuf/changelog#013161

Also there's a confusion between issue numbers and pr numbers on github. I'm not sure if `#1` cannot refer to both...